content/spin/v2/testing-apps.md

```diff
@@ -136,9 +220,9 @@ Before you can run the test, you'll need to tell `spin-test` where your test liv
 $ cd ..
 ```
 
-Then we edit the application's manifest (the `spin.toml` file) as shown below:
+Then we edit the `my-app` application's manifest (the `spin.toml` file) by adding the following block:
 
-<!-- @selectiveCpy -->
+<!-- Leaving out annotation so this can just be copied by the user without any frills -->
```

Comment:
Again... just skip this I think?  If this is something we feel the need to write then let's define a `cpy` block or something, but I vaguely thought it was the default.